Modules/Core/Http/Controllers/BasePublicController.php

```diff
@@ -14,9 +18,67 @@
     protected $auth;
     public $locale;
 
+    /**
+     * @var array Alternate URLs
+     */
+    public $alternateUrls = [];
+
+    /**
+     * @var AssetManager
+     */
+    protected $assetManager;
+    /**
+     * @var AssetPipeline
+     */
+    protected $assetPipeline;
+    /**
+     * @var AssetTypeFactory
+     */
+    protected $assetFactory;
+
     public function __construct()
     {
         $this->locale = App::getLocale();
         $this->auth = app(Authentication::class);
+
+        $this->assetManager = app(AssetManager::class);
+        $this->assetPipeline = app(AssetPipeline::class);
+        $this->assetFactory = app(AssetTypeFactory::class);
+
+        $this->addAssets();
+        $this->requireDefaultAssets();
+        view()->share('alternate', $this->alternateUrls);
+    }
+
+    /**
+     * Add the assets from the config file on the asset manager.
+     */
+    private function addAssets()
```

Comment:
Why all this extra assets stuff? Pretty sure it isn't used in the front end...

Reply:
Same as above, maybe I have some old controller, tomorrow I will remove this request and try to understand whats wring with my branch.
Beside this, have a look to the solution if it can work